paper/paper.md

```diff
@@ -0,0 +1,281 @@
+---
+title: 'Connectome Mapper 3: A Flexible and Open-Source Pipeline Software for Multiscale Multimodal Human Connectome Mapping'
+tags:
+  - Python
+  - connectome
+  - multi-scale
+  - MRI
+  - BIDS
+  - BIDS App
+  - containerized workflows
+  - nipype
+authors:
+  - name: Sebastien Tourbier
+    orcid: 0000-0002-4441-899X
+    affiliation: 1
+  - name: Joan Rue-Queralt
+    orcid: 0000-0001-5680-4540
+    affiliation: 1
+  - name: Katharina Glomb
+    orcid: 0000-0002-4596-4386
+    affiliation: 1
+  - name: Yasser Aleman-Gomez
+    orcid: 0000-0001-6067-8639
+    affiliation: 1
+  - name: Emeline Mullier
+    orcid: 0000-0001-6209-0791
+    affiliation: 1
+  - name: Alessandra Griffa
+    orcid: 0000-0003-1923-1653
+    affiliation: 2
+  - name: Mikkel Schöttner
+    orcid: 0000-0002-4521-9837 
+    affiliation: 1
+  - name: Jonathan Wirsich
+    orcid: 0000-0003-0588-9710
+    affiliation: 3
+  - name: Anil Tuncel
```

Comment:
Can we put this unicode character ('ı') in my name?
It should be like this
`- name: M. Anıl Tuncel`

In latex it's written as {\i}

Thanks :)

Reply:
Yes for sure I will see what I can do